platform/windows/win_resource_file.rc.in

```diff
@@ -0,0 +1,42 @@
+#include "resource.hpp"
```

Comment:
Wrong include path.

Reply:
I removed it. I think we don't need it anyways.